tt_metal/impl/program/program.cpp

```diff
@@ -591,6 +591,37 @@ void Program::set_cb_data_fmt(Device *device, const std::vector<CoreRange> &crs,
     }
 }
 
+void Program::set_cb_tile_dims(Device *device, const std::vector<CoreRange> &crs, JitBuildOptions &build_options) const {
+    ZoneScoped;
+    for (auto logical_cr : crs) {
```

Comment:
auto& in all the range based loops to avoid redundant copies.

Reply:
changed

Reply:
that's a integer access, so should be fine